test/spec/OktaAuth/browser.ts

```diff
@@ -216,10 +216,9 @@ describe('OktaAuth (browser)', function() {
       it('Default options when no refreshToken: will revokeAccessToken and use window.location.origin for postLogoutRedirectUri', function() {
         return auth.signOut()
           .then(function() {
-            expect(auth.tokenManager.getTokensSync).toHaveBeenCalledTimes(3);
             expect(auth.revokeRefreshToken).not.toHaveBeenCalled();
             expect(auth.revokeAccessToken).toHaveBeenCalledWith(accessToken);
-            expect(auth.tokenManager.clear).toHaveBeenCalled();
+            expect(auth.tokenManager.clear).not.toHaveBeenCalled();
```

Comment:
since this assertion is being reversed, can we update the title to include this information?

`Default options: will X, Y, and set tokens to clear after redirect`
 
 add an assert that the pending token renew was saved

Reply:
alternatively, can remove the tokenManager asserts from this test since this logic is covered in another test